src/app/pre-sale/components/pre-sale-widget/index.tsx

```diff
@@ -52,6 +87,13 @@ export function PreSaleWidget({ token }: { token: PreSaleToken }) {
         isLoading={false}
         onClick={onClickButton}
       />
+      {transactionReview && (
+        <TransactionReviewModal
+          isOpen={isTransactionReviewOpen}
+          onClose={onCloseTransactionReview}
+          transactionReview={transactionReview}
+        />
+      )}
```

Comment:
Note that even with the placeholder above it will crash for now, as the popup expects a quote. It does work tho (tested locally). Fill be showing once, quotes are added.